code/datums/abilities/critter/icephoenix.dm

```diff
@@ -0,0 +1,443 @@
+ABSTRACT_TYPE(/datum/targetable/critter/ice_phoenix)
+/datum/targetable/critter/ice_phoenix
+	icon = 'icons/mob/critter/nonhuman/icephoenix.dmi'
+	icon_state = "template"
+
+/datum/targetable/critter/ice_phoenix/sail
+	name = "Sail"
+	desc = "Channel to gain a large movement speed buff while in space for 20 seconds"
```

Comment:
This is the wrong duration, also a 60 second cooldown and a 10 second channel for a minor 10 second movement speed buff seems extremely useless(?)

Reply:
Buffed, LMK if it seems fine